### ISSUE TITLE
segalign didn't run with --gpu option

### ISSUE THREAD

**Issue by lizihe21** [ComparativeGenomicsToolkit/cactus#607]
Hi, i was testing cactus on 4 gpus 64 cores single machine. After cactus finished on the example dataset, I checked the log and found that only segalign_repeatmasker was called. There was no segalign in the log. Is there anything wrong with my config?

**Comment by glennhickey** [comment]
Good catch.  This will be fixed in the next release.  In the meantime, you can work around it by:
```
sed src/cactus/cactus_progressive_config.xml -e  's/gpuLastz="false"/gpuLastz="true"/g' > ./config-gpu.xml
```
then running with `--configFile ./config-gpu.xml`

Note: this bug only affects `cactus` and not `cactus-blast`, nor should it be an issue when using the GPU docker image.
